src/App.css

```diff
@@ -1,38 +1,15 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
+body {
+  margin: 0;
+  font-family: "Oswald", sans-serif;
+  color: #0f1843;
+  text-decoration: none;
+  background-color: #fff4f8;
+}
+
+/*
+orange - #FF7955  (255,121,85)
+pink - #F4327F  (244,50,127)
+purple - #493871 (73,56,113)
+backgroundcolor - #fff4f8
+light pink - #ffc0d7 (255,192,215)
+*/
```

Comment:
If you want these colors to be customisable, you could use [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) for them.

So in `App.css` you define:

```css
:root {
  --orange: #FF7955;
}
```

And then in any other styles you can just use it:
```css
background-color: var(--orange);
```